TankDriver.App/TextureStorage.cs

```diff
@@ -19,13 +19,20 @@ internal class TextureStorage
 		public Texture2D TankTurretTexture { get; private set; }
 
 		/// <summary>
+		/// Bullet texture
+		/// </summary>
+		/// <value>The bullet texture.</value>
+		public Texture2D BulletTexture { get; private set; }
```

Comment:
_You need to be cool™_: remove `private set;` here. Getter-only auto properties are implicitly settable from constructors starting from C# 6.